### DISCUSSION TITLE
参与 Mina 学习激励活动，赢取奖励！

### DISCUSSION THREAD

**Discussion by phoouze** [openbuildxyz/mina-zkapp-bootcamp#154] [Announcements]
**📅 活动时间：11月21日 - 12月30日**

#### 参与途径
**💬 技术讨论**
在学习群或 Git 讨论区积极发言，参与技术探讨并帮助解答问题。
标准：高质量的活跃与互动
奖励：
**课程结束后，根据活跃度和作业提交情况选出前6名，各奖励10美元**。

**✍ 技术文章**
在推特，掘金、V2EX、CSDN、登链、思否或公众号等平台发布技术文章。**须添加 OpenBuild、Mina、zk 等相关标签并推广本次学习激励活动，贴合课程主题**。(需提交文章链接）
标准：高质量技术文章，贴近课程主题，推广活动，添加 OpenBuild、Mina 标签
奖励：
**每发布一篇 500 字以上技术文章立即可得 1 美元创作奖励(⚠️ 禁止刷量），课程结束后根据文章质量与传播效果，选出前6篇优秀文章，每篇奖励20美元**。

**📢 社交分享**
在推特发帖或转帖（需@ **OpenBuildxyz**、**MinaProtocol**、**Mina_Pulse**）或在微信朋友圈转发活动文章。(需提交链接或截图)
标准：高质量社区互动，传播效果
奖励：
**课程结束后，根据发帖质量与互动表现，选出前6名，各奖励10美元**。

**🔔 注意事项**
⚠️可以提交链接或截图在讨论区，也可以进行讨论，不要提交文章内容到讨论区。
技术文章和社交发帖请贴合课程主题。
奖励会以美元等值的 $MINA 发放到 github 提交的钱包中。
有疑问请微信联系小助手：**hahalzr0118**

**🔗 提交**
**文章链接，转帖链接以及活跃截图等请提交到本讨论区。**   🎉 奖励发放情况看[这里](https://github.com/openbuildxyz/mina-zkapp-bootcamp/blob/main/reward/contest.md)

**Comment by lispking** [comment]
已用个人公众号第一时间转发，目前阅读量120+，并拉了一些同学进群学习：https://mp.weixin.qq.com/s/zrTz4c6A8tq_-6p-vkwHQQ

**Comment by MartinYeung5** [comment]
分享一下，自己對ZK技術學習心得，是一整個系列 :
https://ithelp.ithome.com.tw/users/20119569/ironman/8057

**Reply by phoouze** [reply]
也可以在登链发布下，多宣传

**Comment by MartinYeung5** [comment]
在twitter上轉發了活動報名的文章
https://x.com/imsuperkk/status/1859518001575596289
<img src="https://github.com/user-attachments/assets/a2a5b2fd-29b4-4eff-80e6-5b1bdb62df46" alt="Your image title" width="400"/>

**Reply by phoouze** [reply]
顺便提交下链接

**Reply by MartinYeung5** [reply]
已更新了

**Comment by MartinYeung5** [comment]
想問一下，有沒有人用過最新版本的 Protokit ?

**Reply by phoouze** [reply]
@MartinYeung5 我帮你问了，因为新版本和之前版本变化较大，讲师还在熟悉中，后面有资料会分享给你

**Comment by phoouze** [comment]
可以把这个帖子整理成文章，参与活动
https://twitter.com/Mina_Pulse/status/1859451216998301855

**Reply by MartinYeung5** [reply]
整理成文章: https://ithelp.ithome.com.tw/articles/10353736

**Reply by MartinYeung5** [reply]
補充一下，我也在登鏈上發了一篇 : https://learnblockchain.cn/article/10009

**Reply by phoouze** [reply]
奖励已发， https://minascan.io/mainnet/tx/5Ju9kL8RD53QfyUG7T31yhd1vEfrcDfwY7g5ZNSupmtjZnFY2do3

**Comment by tianhuihui1** [comment]
在登链平台发布《零基础开始学 Mina》
https://learnblockchain.cn/article/9990

**Reply by phoouze** [reply]
奖励已发， https://minascan.io/mainnet/tx/5JubrFYmsmNcCoeK1xtxoSAbnd1P3u4PsuCiGwu5tLnzidEXnbfa

**Comment by lispking** [comment]
看到非常有趣的一篇 Mina proof 的文章：[Kimchi：Mina 证明系统的最新更新](https://learnblockchain.cn/article/10013)，顺手翻译成中文

**Reply by phoouze** [reply]
ok，后续文章可以添加 openbuild的标签，顺便再帮助推广这次的课程和活动

**Reply by phoouze** [reply]
奖励已发，https://minascan.io/mainnet/tx/5JuZN4QWTrtxQxY66sEjQrD6ZJcMhnsvvuYWX3aasBG2ABovUsqX

**Comment by lispking** [comment]
新鲜出炉：《重新介绍 Mina：利用“万物证明”构建（真实）物联网》https://learnblockchain.cn/article/10031

**Reply by phoouze** [reply]
泰酷啦

**Reply by phoouze** [reply]
奖励已发，https://minascan.io/mainnet/tx/5JupvnywULNVrmAanLE5yL4Qo8jsgaFnth5pLgonT93d3XwLkYU8

**Comment by lispking** [comment]
新鲜出炉：《Mina协议上的零知识机器学习》 https://learnblockchain.cn/article/10051

**Reply by phoouze** [reply]
补充下公众号文章链接， 不一定在登链发， 其他平台也行

**Reply by lispking** [reply]
好的，收到，已补充：https://mp.weixin.qq.com/s/vAaYppVCfg19mj5w86NCuA

**Reply by phoouze** [reply]
奖励已发，https://minascan.io/mainnet/tx/5JtiynKbPqh34UxQCueYgzgRVNcUQtaWY4bsJHZrkNbgpiLpS3uf

**Comment by longerd** [comment]
在登链社区发布 <Mina Learning - Mina Protocol 中文翻译>  https://learnblockchain.cn/article/10075
git链接 https://github.com/longerd/mina-learning/blob/main/mina-protocol/README.md

**Reply by phoouze** [reply]
不一定在登链发， 其他平台也行， 后续文章可以添加 OpenBuild 和 Mina 的标签，顺便再帮助推广这次的课程和活动

**Reply by phoouze** [reply]
奖励已发，https://minascan.io/mainnet/tx/5JuCZaKreuir6dqSwn1QKQzkz3oGdJPFNzFpB1wjanNpbEx7uWsZ

**Comment by lispking** [comment]
新鲜出炉一篇有趣的白板会议：《Mina 白板会议 TL;DR》https://learnblockchain.cn/article/10079

**Reply by phoouze** [reply]
奖励已发，https://minascan.io/mainnet/tx/5JvEt2Dxab2K1jAHUYQhqAVg18pKijeqrwXX8R9ktSLRMo7Xo3Xx

**Comment by longerd** [comment]
Mina 地址学习 https://learnblockchain.cn/article/10083

**Reply by phoouze** [reply]
奖励已发， https://minascan.io/mainnet/tx/5JuS4PvnXGM8WvREh4oJjqW2RWWpWLAQ7HgnjAakLpfmKTZEjbQn

**Reply by phoouze** [reply]
后续文章请添加 OpenBuild 和 Mina 标签，顺便再帮助推广这次的课程和活动

**Reply by longerd** [reply]
已添加

**Comment by lispking** [comment]
新鲜出炉一篇《Mina 上的 STO 和现实世界资产》：https://learnblockchain.cn/article/10107

**Reply by phoouze** [reply]
奖励已发，https://minascan.io/mainnet/tx/5JuVzt4J9KrRPdTF5PuGzBa54y5fEDvynGUjTeqKjWikSktGiS6A

**Comment by lispking** [comment]
新鲜出炉一篇《Mina: 建立值得信赖的社区》：https://learnblockchain.cn/article/10108

**Reply by phoouze** [reply]
奖励已发， https://minascan.io/mainnet/tx/5Jtbf2ujy56iY82wHpZ1YXqun7usULbdSdcAaEezggqxsWw2Zv4p

**Comment by hello2jie** [comment]
登链《从 0 到 1：实现一个基于零知识证明的寻宝游戏》https://learnblockchain.cn/article/10161

**Reply by phoouze** [reply]
![image](https://github.com/user-attachments/assets/66119742-9d97-4a04-9a91-574c7329295b)
文章违规了

**Reply by hello2jie** [reply]
还在审核中(* /ω＼*)，同步到掘金了 https://juejin.cn/post/7445524780604243978
github地址： https://github.com/hello2jie/SecretHeist/blob/main/tutorial.md

**Reply by phoouze** [reply]
奖励已发， https://minascan.io/mainnet/tx/5JuL1mDujYexVxdnqhSyWDTMYzkMEJ6MetpAozdzBzZatz5fStLf 
后续文章可以帮助推广这次的课程或活动,

**Comment by lispking** [comment]
新鲜出炉一篇《Mina 作为参与实验室》：https://learnblockchain.cn/article/10165

**Reply by phoouze** [reply]
奖励已发，https://minascan.io/mainnet/tx/5JtgdzZxSWJHUanuKVhDCEuP75ESmSRH9ryMoifJDdwexsESE6LF

**Comment by YiShengYouNi** [comment]
总结自己的第一篇MIna学习笔记， Mina账户模型：https://learnblockchain.cn/article/10203

**Reply by phoouze** [reply]
奖励已发， https://minascan.io/mainnet/tx/5JtkVxB7GoPqpMazaW3coGTWa8LLFxajL3xxbAk82RRxMM6WsZTq

**Comment by longerd** [comment]
Mina MerkleTree 学习 https://learnblockchain.cn/article/10209

**Reply by phoouze** [reply]
奖励已发， https://minascan.io/mainnet/tx/5Ju8h467UWyXh9hNRmSpTvisnF39t9HaPfyBCZvePzMnKXcsovS6

**Comment by longerd** [comment]
Mina 开发者常见问题解答翻译  https://learnblockchain.cn/article/10214

**Reply by phoouze** [reply]
奖励已发，https://minascan.io/mainnet/tx/5JvDTSMyUzP9LjiLbiEegsyEJnAHwFueK7cx2UhpL1RnGqerFJyP

**Comment by YiShengYouNi** [comment]
Mina账户初始化：https://learnblockchain.cn/article/10208

**Reply by phoouze** [reply]
奖励已发，https://minascan.io/mainnet/tx/5Jv7qtySrwxnBCqigp3rdMJqZGmMoiBTFbFtGHaaEd1ccXrY1LSD

**Comment by lispking** [comment]
新鲜出炉一篇《Protokit：快速入门》https://learnblockchain.cn/article/10262

**Reply by phoouze** [reply]
奖励已发， https://minascan.io/mainnet/tx/5JvLWigHuG7PBNEqhJWchNMuN3BXU8JN6sQdV9n6pk8Qkij9xhrt

**Comment by lispking** [comment]
新鲜出炉一篇《Protokit 是什么？》：https://learnblockchain.cn/article/10263

**Reply by phoouze** [reply]
奖励已发， https://minascan.io/mainnet/tx/5Jv6YZFP9z7coYXUvsusiUT32heY8aN5VxqbULBy37TouHvWpkuM

**Comment by longerd** [comment]
Auro wallet API https://learnblockchain.cn/article/10321

**Reply by phoouze** [reply]
奖励已发，https://minascan.io/mainnet/tx/5JuRVG7JWvAmR45b3d9QyjPmWyNPcTcQsMz1LCaEuZnQjzZBpQTa

**Comment by longerd** [comment]
Mina 智能合约 https://learnblockchain.cn/article/10276

**Reply by phoouze** [reply]
奖励已发，https://minascan.io/mainnet/tx/5JueT3Tp23Vx7kZgULaENNLzb9qrkQaKiEhxp8GzfJr5BqDmVfsJ

**Comment by lispking** [comment]
一起来看看Mina 2025年的计划，《Mina: 展望 2025 年》：https://learnblockchain.cn/article/10362

**Reply by phoouze** [reply]
奖励已发，https://minascan.io/mainnet/tx/5Jtr7SzgFNyiCT22zStmdx5ujJcPUYn3zLTLXzFmMLGJMudEFaP6

**Comment by lispking** [comment]
今天来学习下Mina与httpz会擦出什么样的火花，《隆重推出“httpz”：值得信赖的互联网》：https://learnblockchain.cn/article/10375

**Reply by phoouze** [reply]
奖励已发，https://minascan.io/mainnet/tx/5JvQZQovWoYzTRkERDCF97xkzE9w9keig6r4xNXz6fBLU1tsLjWe

**Comment by lispking** [comment]
终于找到 Mina 的 Rust 节点源码，今天来看看ReadMe都说了啥，《Open Mina 节点介绍》：https://learnblockchain.cn/article/10392

**Reply by phoouze** [reply]
奖励已发， https://minascan.io/mainnet/tx/5JvCtEMjFMrqhUyRxwh9Ba3vLkseQCJfYkeyJKWyyQaWAUwJj9Xd

**Comment by lispking** [comment]
《OpenMina: 运行非出块和出块节点》https://learnblockchain.cn/article/10420

**Reply by phoouze** [reply]
奖励已发，https://minascan.io/mainnet/tx/5JtheVGWzjWxwUREuHKfXVftvqS17JB9jbCFBVHzYFRPFyrXGAY1

**Comment by lispking** [comment]
《OpenMina: 扫描状态》https://learnblockchain.cn/article/10421

**Reply by phoouze** [reply]
奖励已发，https://minascan.io/mainnet/tx/5JtqCcxiyTFTAJ3pETEwnYrLtnPKmx3vPHrwMjYiNArEgfzmQEon

**Comment by lispking** [comment]
《OpenMina: 提交和生成新的 SNARK》https://learnblockchain.cn/article/10422

**Reply by phoouze** [reply]
奖励已发， https://minascan.io/mainnet/tx/5JvK2F5xyeR86oBsuxb5vD7jHvTgzVeoxtd9NGPaJz2fuYkzAjdF

**Comment by lispking** [comment]
《OpenMina: 测试用例说明》：https://learnblockchain.cn/article/10436

**Reply by phoouze** [reply]
奖励已发，https://minascan.io/mainnet/tx/5JvJsMMaxXsatipeocwQpKaFYX8gvqyqryNNFb1RGdhscwvTvVs9

**Comment by lispking** [comment]
《Mina的测试框架》：https://learnblockchain.cn/article/10437

**Reply by phoouze** [reply]
奖励已发，https://minascan.io/mainnet/tx/5JtsqCRduyxMsCBvHSVhC6jrn1TaSpKJpd6dUN3xWRszGwQVCW4h
